lib/parity/environment.rb

```diff
@@ -50,11 +50,23 @@ def restore
         Backup.new(
           from: arguments.first,
           to: environment,
-          additional_args: arguments.drop(1).join(" ")
+          additional_args: additional_restore_arguments,
         ).restore
       end
     end
 
+    def additional_restore_arguments
+      (arguments.drop(1) + [restore_confirmation_argument]).
+        compact.
+        join(" ")
+    end
+
+    def restore_confirmation_argument
+      if !["production", "development"].include?(environment)
```

Comment:
I'm curious why you added development to the protected list?

Also, thoughts on extracting this array to a constant? Maybe `PROTECTED_ENVIRONMENTS` ?